### PR TITLE
Modify/datetime

### PR DESCRIPTION
**Pull request by lpoli**
Modify system tests w.r.t. changes in PR https://github.com/0chain/blobber/pull/748.

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | modify/datetime  |
| gosdk  | modify/datetime  |
| zboxcli  | modify/datetime  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | modify/datetime  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | modify/datetime  |
| gosdk  | modify/datetime  |
| zboxcli  | modify/datetime  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | modify/datetime  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: tests/cli_tests/zboxcli_create_dir_test.go [hunk 1]
```diff
@@ -109,7 +109,8 @@ func TestCreateDir(t *testing.T) {
 		output, err := createDir(t, configPath, allocID, longDirName, false)
 		require.NotNil(t, err, "expected create dir failure command executed with output: ", strings.Join(output, "\n"))
 		require.Len(t, output, 1)
-		require.True(t, strings.HasPrefix(output[0], `CreateDir failed:  {"error":"ERROR: value too long for type character varying(100) (SQLSTATE 22001)"}`), "expected create dir failure command executed with output: ", strings.Join(output, "\n"))
+		aggregatedOutput := strings.Join(output, " ")
+		require.Contains(t, aggregatedOutput, "consensus failed")
```
[the remark this answers is not in the document]
Comment (lpoli) [reply tests/cli_tests/zboxcli_create_dir_test.go:113]: Actually I thought about it. But you know, when blobbers are rejecting commit request, it means consensus is not met.
If an allocation would have 128 blobbers, then there will be 128 messages. Gosdk can't figure out which error to consider for.

Maybe we could collect common messages among blobbers and return to the user with large count.
Previous directory creation process was incomplete.

[the remark this answers is not in the document]
Comment (stewartie4) [reply tests/cli_tests/zboxcli_create_dir_test.go:113]: "when blobbers are rejecting commit request, it means consensus is not met."
based on this, let's translate the error message to "directory creation failed" - something a bit more meaningful to the end user

**Review by stewartie4 — CHANGES_REQUESTED** [review]
agree with Kishan - a better error message is needed, even if it is just create failed

File: tests/cli_tests/zboxcli_create_dir_test.go [hunk 2]
```diff
@@ -349,7 +322,8 @@ func TestCreateDir(t *testing.T) {
 		output, err = createDirForWallet(t, configPath, nonAllocOwnerWallet, true, allocID, true, "/mydir", false)
 		require.NotNil(t, err, "Expected create dir failure but got output: ", strings.Join(output, "\n"))
 		require.Len(t, output, 1)
-		require.True(t, strings.HasPrefix(output[0], `CreateDir failed:  {"code":"invalid_signature","error":"invalid_signature: Invalid signature"}`), "Expected create dir failure but got output: "+strings.Join(output, "\n"))
+		aggregatedOutput := strings.Join(output, " ")
+		require.Contains(t, aggregatedOutput, `consensus not met`)
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_create_dir_test.go:326]: Same as above

**Pull request merged**